app/api.py

```diff
@@ -300,7 +308,7 @@ async def all_exception_handler(request: Request, exc: Exception):
             "data": {"error": str(exc)},
             "message": "server error",
         },
-    )
+    )'''
```

Comment:
Solid choices on the error messaging and overall docstring disambiguation. Although for my money triple double-quotes are the way to go. Triple singles only feel half as awesome.